sentinel-core/src/test/java/com/alibaba/csp/sentinel/eagleeye/EagleEyeCoreUtilsTest.java

```diff
@@ -0,0 +1,180 @@
+package com.alibaba.csp.sentinel.eagleeye;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class EagleEyeCoreUtilsTest {
+
+    @Rule
+    public final ExpectedException thrown = ExpectedException.none();
+
+    @Test
+    public void testIsBlank() {
+        Assert.assertTrue(EagleEyeCoreUtils.isBlank(""));
+        Assert.assertTrue(EagleEyeCoreUtils.isBlank(" "));
+        Assert.assertTrue(EagleEyeCoreUtils.isBlank(null));
+
+        Assert.assertFalse(EagleEyeCoreUtils.isBlank("foo"));
+    }
+
+    @Test
+    public void testCheckNotNullEmpty() {
+        Assert.assertEquals("foo",
+                EagleEyeCoreUtils.checkNotNullEmpty("foo", "bar"));
+    }
+
+    @Test
+    public void testCheckNotNullEmptyInputEmptyStringThrowsException() {
+        thrown.expect(IllegalArgumentException.class);
+        EagleEyeCoreUtils.checkNotNullEmpty("", "bar");
+        // Methods are not expected to return due to exception thrown
+    }
+
+    @Test
+    public void testCheckNotNullEmptyIntputSpaceThrowsException() {
```

Comment:
Typo in test method name `testCheckNotNullEmptyIntputSpaceThrowsException` (Intput)

Reply:
I missed that... Thanks